lib/srv/db/common/kerberos/kinit/kinit.go

```diff
@@ -297,10 +346,18 @@ func (k *CommandLineInitializer) UseOrCreateCredentials(ctx context.Context) (*c
 		return nil, nil, trace.Wrap(cmd.Err)
 	}
 
-	cmd.Env = append(cmd.Env, []string{fmt.Sprintf("%s=%s", krb5ConfigEnv, krbConfPath)}...)
+	cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", krb5ConfigEnv, krbConfPath))
+	cmd.Env = append(cmd.Env, "KRB5_TRACE=/dev/stdout")
```

Comment:
nit: Since the tracing information may be large, should we consider splitting it into a separate log message (setting it to debug)?

Reply:
I think being verbose is fine in this particular instance. The errors are not expected too often and are tricky to diagnose, so I'd rather make it easier to spot the problem early.